corehq/apps/domain/calculations.py

```diff
@@ -377,18 +377,14 @@ def calced_props(dom, id, all_stats):
     }
 
 
-def total_distinct_users(domains=None):
+def total_distinct_users(domain):
```

Comment:
Don't you also need to change the call? https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/domain/calculations.py#L342